clock-bound-c/src/lib.rs

```diff
@@ -95,7 +95,6 @@ pub struct ResponseAfter {
 }
 
 /// A structure for holding the response of a timing request.
-#[derive(Debug)]
```

Comment:
Why remove this? Everything which can implement `Debug`, should. Likewise `Copy` and `Clone`.

Reply:
Can add it back, I had it there for testing but then removed it